.github/workflows/main.yml

```diff
@@ -60,8 +60,8 @@ jobs:
           flags: core
           directory: ./packages/core
 
-      - name: Codecov ui
+      - name: Codecov connector
         uses: codecov/codecov-action@v3
         with:
-          flags: ui
-          directory: ./packages/ui
+          flags: connector
+          directory: ./packages/connector
```

Comment:
actually should remove this section?

Reply:
We might also add test cases for connector kit in the future, so I think it's OK for now.